docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -194,6 +196,60 @@ command completion, or environment variables are presented throughout the
 Ansible Collection Testing because the act of installing the stable release of
 Ansible containing `ansible-test` is expected to setup those things for you.
 
+.. _meta_runtime_yml:
+
+meta directory
+--------------
+
+A collection can store some additional metadata in a ``runtime.yml`` file in the collection's ``meta`` directory. The ``runtime.yml`` file supports the top level keys:
+
+- *requires_ansible*:
+
+  The version of Ansible required to use the collection.
+
+  .. code:: yaml
+
+     requires_ansible: ">=2.10"
+
+- *plugin_routing*:
+
+  A mapping of content in a collection that has been relocated or deprecated.
+  The top level keys of ``plugin_routing`` are types of plugins (for example, ``become``, ``connection``, and ``inventory``).
+  The individual resources under those types can define ``redirect`` for a new location and ``deprecation`` and ``tombstone`` to define a warning message and removal date or version.
```

Comment:
Is there a difference between `deprecation` and `tombstone`?

Reply:
I believe `tombstone` is intended to be used when a plugin is actually removed after the deprecation period.

Reply:
`deprecation` generates a runtime warning (unless disabled by `DEPRECATION_WARNINGS` config) about plugins and/or `redirect`s that will be removed in future releases. eg, a routing entry that contains both `deprecation` and `redirect` will issue a warning when the plugin is accessed via that routing entry. `tombstone` generates a fatal error for plugins that have already been removed. Plugins that have merely been moved elsewhere should use `redirect` not `tombstone`, as it's a fatal error; it should only be used for a plugin that's truly dead (eg, during collections search, we don't keep looking if we hit a `tombstone` entry).